ajax/libs/just-animate/0.6.0/JustAnimate.min.js

```diff
@@ -0,0 +1,2 @@
+"use strict";var Helpers_1=require("./core/Helpers"),Transformers_1=require("./core/Transformers"),ElementAnimator_1=require("./core/ElementAnimator"),SequenceAnimator_1=require("./core/SequenceAnimator"),TimelineAnimator_1=require("./core/TimelineAnimator"),DEFAULT_ANIMATIONS=[],JustAnimate=function(){function e(){var e=this;this._registry={},Helpers_1.each(DEFAULT_ANIMATIONS,function(r){return e._registry[r.name]=r})}return e.inject=function(e){Array.prototype.push.apply(DEFAULT_ANIMATIONS,Helpers_1.map(e,Transformers_1.animationTransformer))},e.prototype.animate=function(e,r,n){return new ElementAnimator_1.ElementAnimator(this,e,r,n)},e.prototype.animateSequence=function(e){return new SequenceAnimator_1.SequenceAnimator(this,e)},e.prototype.animateTimeline=function(e){return new TimelineAnimator_1.TimelineAnimator(this,e)},e.prototype.findAnimation=function(e){return this._registry[e]||void 0},e.prototype.register=function(e){return this._registry[e.name]=Transformers_1.animationTransformer(e),this},e}();exports.JustAnimate=JustAnimate;
+//# sourceMappingURL=JustAnimate.min.js.map
```

Comment:
I should have added this in the Issue:

The main folder from GitHub is https://github.com/just-animate/just-animate/tree/master/dist/browser **except the npm-debug.log which shouldn't be there.**  

Files:

- just-animate-animations.js
- just-animate-animations.min.js
- just-animate-core.js
- just-animate-core.min.js
- just-animate-systemjs.js
- just-animate-systemjs.min.js